funding/manager.go

```diff
@@ -4603,6 +4622,18 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 		scidFeatureVal = true
 	}
 
+	// At this point, if we have an AuxFundingController active, we'll check
+	// to see if we have a special tapscript root to use in our MuSig2
+	// funding output.
+	tapscriptRoot, err := deriveTapscriptRoot(
+		f.cfg.AuxFundingController, chanID,
+	)
+	if err != nil {
+		err = fmt.Errorf("error deriving tapscript root: %w", err)
+		msg.Err <- err
```

Comment:
just `msg.Err <- fmt.Errorf(...`

Reply:
Actually, I forgot to do the logging here, which I think we need. So I kept the separate variable.